### ISSUE TITLE
Revisit channel pages & authentication

### ISSUE THREAD

**Issue by tmbo** [RasaHQ/rasa#7254]
**Description of Problem**: Some of the current channel documentation pages are hard to follow and setting them up isn't exactly easy:

In addition to that, we should verify we are using security best practices for all of these channels, e.g. validating signatures to ensure the caller is who we expect they are (or e.g. using random webhook URLs as suggested by Telegram).

**Examples** (if relevant):

*   <span class="error">[x]</span> Slack (already fixed)
*   [ ] Facebook messenger (complicated )
*   [ ] Twilio (maybe add screenshots?)
*   [ ] hangouts (very wordy, no steps)

**Definition of Done**:

*   [ ] Documentation is updated
*   [ ] Security is validated for each channel

**Comment by m-vdb** [comment]
@tmbo this issue seems to be 2 distinct issues 1) improve the docs 2) improve security in our channels. Would you care to split it?
